examples/concepts/cursors.html

```diff
@@ -19,38 +19,26 @@
   }
 </style>
 
-<p>By default, cursors are disabled. To enable them, set the `showSVGCursors`
-  option to `true` when initializing `cornerstoneTools`.</p>
-
-{% highlight javascript %}
-cornerstoneTools.init({
-  showSVGCursors: true,
-});
-{% endhighlight %}
+<p>
+  By default, cursors are disabled. To enable them, set the `showSVGCursors`
+  option to `true` when initializing `cornerstoneTools`.
+</p>
 
+{% highlight javascript %} cornerstoneTools.init({ showSVGCursors: true, }); {%
+endhighlight %}
```

Comment:
I get that these are probably prettier/eslint auto-fixes, but we'll almost always want formatting to be preserved here. Best to wrap highlight blocks in ignore tags